thetis/options.py

```diff
@@ -417,14 +443,17 @@ class NonhydrostaticModelOptions(FrozenHasTraits):
     q_degree = NonNegativeInteger(
         None, allow_none=True, help="Polynomial degree of the non-hydrostatic pressure space").tag(config=True)
     update_free_surface = Bool(True, help='Update free surface elevation after pressure projection/correction step').tag(config=True)
-    solver_parameters = PETScSolverParameters({
```

Comment:
Why are `solver_parameters` disappearing? There should be two independent sets of `solver_parameters` here: 1) this one for the NH Poisson solve and 2) the one under the free surface timestepper. 1) is not associated with any timestepper because it's not got a time derivative